### DISCUSSION TITLE
Add this program to winget's repository

### DISCUSSION THREAD

**Discussion by joaogelado** [ThioJoe/YT-Spammer-Purge#144] [Ideas]
I just worder: can we add this program onto the winget's repo so we can download and upgrade the script from one command?

**Comment by TechStudent10** [comment]
Well, yeah it's possible. All you just need is an installation script. I can make one easily, but the problem is getting it in the PATH. You'll need to be able to add the script to the PATH. That's the only problem.

**Reply by joaogelado** [reply]
Can't you run as administrator through the UAC? I really have no idea how to do UAC through cmd but should be possible.

**Reply by TechStudent10** [reply]
@joaogelado idk you'll have to look through the MS docs. But what does running as admin has to do with it.

**Reply by joaogelado** [reply]
I was thinking you couldn't add to the PATH because of administrator privileges required

**Reply by joaogelado** [reply]
And adding the path should be easy no? I was thinking of "set PATH = %PATH%;C:/Program/Path/Here", but I don't really know

**Reply by TechStudent10** [reply]
Hm, let me do some research.

**Reply by Trenly** [reply]
Winget can't support portable applications at this time. It is planned for a future release, but isn't out yet. In order for this to be added to winget, the executable would have to be packaged inside of an installer. It should be possible to set up Nullsoft or Inno to build the installer, but that is something that would be required before this can be added to winget

**Reply by TechStudent10** [reply]
@Trenly I didn't know that.

**Comment by A-Metal-Penguin** [comment]
> I just worder: can we add this program onto the winget's repo so we can download and upgrade the script from one command?

I second this idea!
